Game/src/Player.java

```diff
@@ -189,10 +185,16 @@ public void onUpdate(boolean up, boolean left, boolean right) {
         }
 
         // TODO : Update player falling.
-        if( !inJumpAnimation && !playerCollision.isCollidingBottom(box, this) && !checkStageCollisionBottom()){
+        if( !inJumpAnimation && !playerCollision.isCollidingBottom(MainApplication.box2, this) && !checkStageCollisionBottom()){
             inJumpAnimation = true;
 
         }
+        if(playerCollision.isCollidingBottom(MainApplication.box2, this)){
+            vY=0;
+            inJumpAnimation = false;
+        }
+        System.out.println(inJumpAnimation);
```

Comment:
Any print statements should be marked with a `// TODO: Debug Print Statement, Remove Later` that way they are easy to find when we have to remove them